bin/rucio-admin

```diff
@@ -1003,18 +1004,55 @@ def declare_bad_file_replicas(args):
         for pfn in bad_files_pfns:
             print(pfn)
 
-    chunk_size = 10000
-    tot_files = len(bad_files)
-    cnt = 0
-    nchunk = math.ceil(tot_files / chunk_size)
-    for chunk in chunks(bad_files_pfns, chunk_size):
-        cnt += 1
-        client.add_bad_pfns(pfns=chunk, reason=args.reason, state='BAD', expires_at=None)
-        ndeclared = len(chunk)
-        print('Chunk %s/%s : %s replicas successfully declared' % (int(cnt), int(nchunk), ndeclared))
-    print('--------------------------------')
-    print('Summary')
-    print('%s replicas successfully declared' % tot_files)
+    if len(bad_files_pfns) < 100:
+        # Using the old API to declare
+        non_declared = client.declare_bad_file_replicas(pfns=bad_files_pfns, reason=args.reason)
+        for rse in non_declared:
+            for pfn in non_declared[rse]:
+                print('%s : PFN %s cannot be declared.' % (rse, pfn))
+    else:
+        print('Getting the information about RSE protocols. It can take several seconds')
+        dict_rse = client.export_data(distance=False)
```

Comment:
Has the export feature been tested? For one thing, the appropriate permission is not defined, so it defaults to `_is_root()`. For another, it didn’t work for me.
```
>>> from rucio.client import Client
>>> c = Client(account='ddmadmin')
>>> d = c.export_data(distance=False)
2021-04-07 13:15:21,720 WARNING Waiting 0.25s due to reason: server returned 502 
2021-04-07 13:20:22,381 WARNING Waiting 0.5s due to reason: server returned 502 
2021-04-07 13:25:23,212 WARNING Waiting 1.0s due to reason: server returned 502 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/var/home/dchristidis/.local/lib/python3.9/site-packages/rucio/client/exportclient.py", line 54, in export_data
    raise exc_cls(exc_msg)
rucio.common.exception.RucioException: An unknown exception occurred.
Details: no error information passed (http status code: 502)
```

Reply:
Yes, I noticed the missing permission. It works on the dev container

Reply:
I don't think we should use the export API here. That's really not the use-case it was made for. Can't you use rse_info?

Reply:
How ?? We have no reverse method to get a RSE from a hostname.

Reply:
As discussed offline, let's go ahead with it like this.

Reply:
Issue still stands. I am afraid I cannot approve this dependence on the export API until I can confirm that it works. It appears that, in addition to the problems described in my previous comment, the endpoint might not be properly registered on Flask.
```
>>> from rucio.client import Client
>>> c = Client(account='ddmadmin')
>>> d = c.export_data(distance=False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/var/home/dchristidis/.local/lib/python3.9/site-packages/rucio/client/exportclient.py", line 54, in export_data
    raise exc_cls(exc_msg)
rucio.common.exception.RucioException: An unknown exception occurred.
Details: no error information passed (http status code: 404)
```

Reply:
@cserf can you please check if the endpoint is registered? A testcase would of course help here to show these errors 😏

Reply:
I think there is a misconception here. As far as I can see the endpoint is well-defined. It needs to be configured in the server's rucio.cfg though:

```
[api]
endpoints = …, export, …
```

This is already done in the latest dev container, so make sure you have that if you're testing this.
https://github.com/rucio/containers/blob/master/dev/rucio.cfg#L183

Reply:
The 502 error was due to a problem with the exporter API in Flask. It was working fine on web.py :(
Fixed by https://github.com/rucio/rucio/pull/4658

Reply:
Rebase + Fix permission